Gordon360/Services/HousingService.cs

```diff
@@ -505,7 +506,18 @@ public ApartmentApplicationViewModel GetApartmentApplication(int applicationID,
         /// <returns>Array of ApartmentApplicationViewModels</returns>
         public ApartmentApplicationViewModel[] GetAllApartmentApplication()
         {
-            return _context.Housing_Applications.AsEnumerable().Select(a => GetApartmentApplication(a.HousingAppID, true)).ToArray();
+            var enumerable = _context.Housing_Applications.AsEnumerable();
+            var currentSession = Helpers.GetCurrentSession(_context);
+
+            // TO DO: Refactor Housing App so that the application itself is connected
+            //    to a session rather than the applicants!
+            enumerable = enumerable.Where(a => _context.Housing_Applicants.Where(
```

Comment:
I'm not 100% sure why the foreign key relationship was not working (i.e. `a.Housing_Applicants.Any(`) instead of the _context call, but this was the only way it worked.

Reply:
You have to call `Include` for the navigation property to work.

Reply:
Presumably that does the same exact thing as this, right? Not a blocking issue?